src/app/migrations/0002_auto_20230524_0624.py

```diff
@@ -0,0 +1,24 @@
+# Generated by Django 3.2.18 on 2023-05-24 06:24
```

Comment:
Commit only the relevant migration files. Migration run on local changes also create these files. 
Removing all these and then rerunning migrations should fix this. Or checkout to master and create migration again in this branch.